spring-cloud-zookeeper-discovery/pom.xml

```diff
@@ -16,6 +16,22 @@
 		<relativePath>..</relativePath>
 	</parent>
 
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>org.codehaus.gmavenplus</groupId>
+				<artifactId>gmavenplus-plugin</artifactId>
+			</plugin>
+            <plugin>
```

Comment:
ok maybe I'll do a single commit right now that will convert all the spaces to tabs? WDYT?